creator-node/src/dbManager.js

```diff
@@ -468,6 +469,35 @@ class DBManager {
     return DBManager._getLegacyStoragePathsAndCids(cursor, batchSize, true)
   }
 
+  static async getCustomStoragePathsRecords(cursor, batchSize) {
+    const queryResult = await models.File.findAll({
+      attributes: [
+        'storagePath',
+        'multihash',
+        'dirMultihash',
+        'fileName',
+        'trackBlockchainId',
+        'fileUUID'
+      ],
+      where: {
+        multihash: { [sequelize.Op.gte]: cursor },
+        storagePath: {
+          [sequelize.Op.notLike]: `${config.get('storagePath')}/%`,
+          [sequelize.Op.notLike]: '/file_storage/%'
+        }
+      },
+      order: [['multihash', 'DESC']],
+      limit: batchSize
+    })
+    logger.debug(
+      `queryResult for custom storagePaths with cursor ${cursor}: ${JSON.stringify(
+        queryResult || {}
+      )}`
+    )
+    if (isEmpty(queryResult)) return []
+    return queryResult
+  }
+
   static async updateLegacyPathDbRows(copiedFilePaths, logger) {
     if (!copiedFilePaths?.length) return true
     const transaction = await models.sequelize.transaction()
```

Comment:
every row is updated sequentially so should only be one connection at a time? my concern is that if this fails, it rollsback everything, unnecessarily, and then in the outer code we don't retry

we could make this code more robust but it doesnt really matter since next loop will run it again, doesnt really matter. removing tx prob simpler but nbd

Reply:
discussed offline, doesnt matter - resolving